apis/server/volume_bridge.go

```diff
@@ -137,3 +140,46 @@ func (s *Server) removeVolume(ctx context.Context, rw http.ResponseWriter, req *
 	rw.WriteHeader(http.StatusNoContent)
 	return nil
 }
+
+func (s *Server) pruneVolume(ctx context.Context, rw http.ResponseWriter, req *http.Request) error {
+	var volumePruneRep types.VolumePruneResp
+
+	filter, err := filters.FromParam(req.FormValue("filters"))
+	if err != nil {
+		return err
+	}
+
+	volumeList, err := s.VolumeMgr.List(ctx, filter)
+	if err != nil {
+		return err
+	}
+
+	for _, volume := range volumeList {
+		containers, err := s.ContainerMgr.List(ctx, &mgr.ContainerListOption{
```

Comment:
could we get the volume not being used in one time, since we can use the filterfunc like `if v.Namenot in volumes` ?